Packages/com.unity.inputsystem/InputSystem/Editor/Internal/InputStateWindow.cs

```diff
@@ -330,18 +410,31 @@ private static class Styles
                 fontSize = EditorStyles.boldFont.fontSize - 2,
                 normal = new GUIStyleState { textColor = Color.black }
             };
+
+            public static GUIStyle hexLabel = new GUIStyle
+            {
+                fontStyle = FontStyle.Normal,
+                font = Font.CreateDynamicFontFromOSFont("Courier", EditorStyles.label.fontSize + 2),
```

Comment:
Related to comment of Courier in sync, see https://www.grputland.com/2012/08/font-stacks-that-look-similar-in.html#cour20120806 for platform specific options for linux, mac, win if there is a desire to use something else. However I think this is fine as is and I think its the right call to align with other editor monospaced representations if there is nothing bundled.